react/features/notifications/functions.js

```diff
@@ -15,3 +17,12 @@ export function areThereNotifications(stateful: Object | Function) {
 
     return enabled && notifications.length > 0;
 }
+
+/**
+ * Tells wether join/leave notifications are enabled in interface_config.
+ *
+ * @returns {boolean}
+ */
+export function joinLeaveNotificationsDisabled() {
+    return Boolean(interfaceConfig && interfaceConfig.DISABLE_JOIN_LEAVE_NOTIFICATIONS);
```

Comment:
We can use optional chaining now, so this can be written as `Boolean(interfaceConfig?.DISABLE_JOIN_LEAVE_NOTIFICATIONS)`

Reply:
Nice!